modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -38,6 +38,7 @@
                 :current-fields="currentFields"
                 :trigger-validation="triggerValidation"
                 :utility-rail="false"
+                :url="doc.data._url"
```

Comment:
the (in)famous `url` in `AposDocEditor`

Reply:
This doesn't seem like it's going to work. It's probably related to why I can't see the attachment slat in the article doc editor, since it's trying to pass in the article's URL.

Reply:
That might be related to what I described: "if you add an attachment to a schema, it will upload and the file will be displayed in the "Files" modal, but not in the document you added it into. If you look at the db, the attachment field of your doc is null". Does it work in "Media/Files" though?

Reply:
Attachment URLs and document URLs are not related in any way. This should not be passed in, at least not for the sake of attachment editing.

Reply:
It looks like the slats aren't visible in the Articles context because there is no `_id` on the slat item to serve as the `:key`. When I resolve that the slat appears but is empty. Something is going on that those attachment items aren't fully populated.